File: OverlayManager/TS/DragonH/CurseBlessingDisease.ts
class CurseBlessingDisease {
  name: string;
  description: string;
  mods: Array<Mod> = Array<Mod>();
  duration: TimeSpan = TimeSpan.infinity;
  releaseTriggers: Array<ReleaseTrigger> = Array<ReleaseTrigger>();
  constructor() {

  }
}

File: DHDM/DnD Model/CurseBlessingDisease.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace DHDM
{
	public class CurseBlessingDisease
	{
		public string name;
		public string description;
		public List<Mod> mods = new List<Mod>();
		public List<ReleaseTrigger> releaseTriggers = new List<ReleaseTrigger>();

		public DndTimeSpan duration = DndTimeSpan.Infinity;
		CurseBlessingDisease()
		{

		}
	}
}
